ui-ngx/src/app/shared/components/time/datetime-period.component.ts

```diff
@@ -43,7 +43,6 @@ export class DatetimePeriodComponent implements OnInit, ControlValueAccessor {
 
   maxStartDate: Date;
   minEndDate: Date;
-  maxEndDate: Date;
```

Comment:
Bunu neden kaldırdık?

Reply:
Dashboard'ta tahmin verilerini göstermek için yapmıştık diye hatırlıyorum